client/packages/common/src/intl/locales/en/common.json

```diff
@@ -858,7 +861,7 @@
   "label.import-cce": "Import cold chain equipment",
   "label.import-facility-properties": "Import facility properties",
   "label.in-packs-of": "in packs of",
-  "label.in-stock-units": "In stock (units)",
+  "label.in-stock": "in Stock",
```

Comment:
Hard to tell why this is capitalised? Perhaps in-stock-units, but as "{{unitName}} in Stock" would be better?

Reply:
Wanna keep without unit name to keep translation function going since we have places where it's more implicit (in inbound where we don't say {{unitName}} received) while others are explicit (in prescriptions where it's {{unitName/doses}} in stock)

Reply:
But what if that doesn't make sense in other language (e.g. in english its "doses in stock" but can imagine different grammar, might want "in stock doses")?